jobs/twitter_user.rb

```diff
@@ -1,27 +1,26 @@
 #!/usr/bin/env ruby
-require 'net/http'
+require 'twitter'
 
 # Track public available information of a twitter user like follower, follower
 # and tweet count by scraping the user profile page.
 
 # Config
 # ------
+
 twitter_username = ENV['TWITTER_USERNAME'] || 'foobugs'
 
-SCHEDULER.every '2m', :first_in => 0 do |job|
-  http = Net::HTTP.new("twitter.com", Net::HTTP.https_default_port())
-  http.use_ssl = true
-  response = http.request(Net::HTTP::Get.new("/#{twitter_username}"))
-  if response.code != "200"
-    puts "twitter communication error (status-code: #{response.code})\n#{response.body}"
-  else
+Twitter.configure do |config|
+  config.consumer_key = YOUR_CONSUMER_KEY
+  config.consumer_secret = YOUR_CONSUMER_SECRET
+  config.oauth_token = YOUR_OAUTH_TOKEN
+  config.oauth_token_secret = YOUR_OAUTH_TOKEN_SECRET
+end
 
-    tweets = /profile["']>[\n\t\s]*<strong>([\d.]+)/.match(response.body)[1].delete('.').to_i
-    following = /following["']>[\n\t\s]*<strong>([\d.]+)/.match(response.body)[1].delete('.').to_i
-    followers = /followers["']>[\n\t\s]*<strong>([\d.]+)/.match(response.body)[1].delete('.').to_i
-    
-    send_event('twitter_user_tweets', current: tweets)
-    send_event('twitter_user_followers', current: followers)
-    send_event('twitter_user_following', current: following)
-  end
-end
\ No newline at end of file
+twitter_username = ENV['TWITTER_USERNAME'] || 'foobugs'
```

Comment:
yes it is, as I’m no going to accept this pull request as it needs to have oauth keys configured which is more complicated than just adding the username